gitpod-io/gitpod
# [public-api] cell stats initial

## Description


## Related Issue(s)

Fixes SID-295

## How to test


1. `kubectl port-forward service/server 9877:9877`
2. `curl --header "Content-Type: application/json" --data '{}' http://localhost:9877/gitpod.experimental.v1.StatsService/GetUserStats`

You should see something like
```json
{"registeredUsers":1}
```

## Documentation


#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - ak-cell-stats-api</li>
	<li><b>🔗 URL</b> - <a href="https://ak-cell-stats-api.preview.gitpod-dev.com/workspaces" target="_blank">ak-cell-stats-api.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
</ul>

## Build Options

<details>
<summary>Build</summary>

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

</details>

<details>
<summary>Installer</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>
 
/hold
